locales/en/pin_unlock_help.md

```diff
@@ -1,9 +1,7 @@
-Il PIN è il codice di sicurezza che mantiene nell'app IO i tuoi dati protetti.
+The PIN is the security code that grant your data being securely stored into the IO app.
 
-Lo hai scelto quando sei entrato per la prima volta nell'app tramite la tua identità SPID.
+You chose it when you logged in for the first time with your SPID credentials.
 
-Se non lo ricordi, puoi modificarlo in ogni momento  semplicemente premendo il pulsante in basso e usando nuovamente la tua identità SPID.
+If you forgot it, you can change it at any time by pressing the "Did you forget the PIN?" button on the bottom and logging in again by your SPID credentials.
```

Comment:
```suggestion
If you forgot it, you can change it at any time by pressing the "Did you forget the PIN?" button on the bottom and logging in again with your SPID credentials.
```

Reply:
I'm not sure about the usage of "by" with login. Looking on transactions it seems to be more appropriate the usage of "with"

Reply:
@valentinacala 
> I'm not sure about the usage of "by" with login. Looking on transactions it seems to be more appropriate the usage of "with"

So it looks like better as I suggested? Anyway I guess it's correct in both ways

Reply:
Yes, it seems to be. committed!